app/vmui/packages/vmui/config/plugins/dynamicIndexHtml.ts

```diff
@@ -0,0 +1,26 @@
+import { readFile } from "fs/promises";
+import { IndexHtmlTransform } from "vite";
+
+/**
+ * Vite plugin to dynamically load index.html based on the current mode.
+ * If a specific mode-based index file (e.g., index.victorialogs.html) exists, it is used.
+ * Otherwise, the default index.html is loaded.
+ */
+export default function dynamicIndexHtmlPlugin({ mode }) {
+  return {
+    name: "vm-dynamic-index-html",
+    transformIndexHtml: {
+      order: "pre",
+      handler: async () => {
+        try {
+          return await readFile(`./index.${mode}.html`, "utf8");
```

Comment:
nit. `readFile` - can return the error that we ignore. I think it is better to handle this erro